Sources/GDSAnalytics/Screens/ScreenParameter.swift

```diff
@@ -3,5 +3,7 @@ public enum ScreenParameter: String {
     case endpoint
     case status
     case hash
-    case title
+    case title = "screen_name"
```

Comment:
As above I think firebase already provide these names as constants for `screen_name`, `screen_id` and `screen_class` not sure if we should be passing our own string

Reply:
💯 % this